data-raw/process_data_sets.R

```diff
@@ -272,3 +272,14 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+
+# Avocado Prices By Region
+# Original Google Sheet here:
+# https://docs.google.com/spreadsheets/d/1cNuj9V-9Xe8fqV3DQRhvsXJhER3zTkO1dSsQ1Q0j96g/edit#gid=1419070688
+avocados <-
+  "https://wjhopper.github.io/SDS-201/data/avocados.csv" %>%
+  read_csv() %>%
+  janitor::clean_names()
```

Comment:
You can remove the explicit call to `janitor::` since we load it at the beginning of the script